kuma/wiki/templatetags/jinja_helpers.py

```diff
@@ -263,3 +264,32 @@ def wiki_url(path):
     if fragment:
         new_path += '#' + fragment
     return new_path
+
+
+def get_bare_tag(elem):
+    """
+    Returns the tag without the leading namespace
+    """
+    return elem.tag.rsplit('}', 1)[-1]
+
+
+@library.global_function
+def include_svg(path, title=None):
```

Comment:
I'm a little worried about the performance - this is called 5x per page, and uses some potentially expensive methods - but premature optimization is usually a bad idea. It should be easy to add Django's ``@lru_cache``, or look for in-code optimizations, if it looks problematic.

I do want some tests though, such as:

* ``include_svg('includes/icons/social/twitter.svg')``
* ``include_svg('includes/icons/social/twitter.svg', 'New Title')``
* ``include_svg('includes/icons/social/twitter.svg', 'Nuevo Título')``

Tests can be hard to write - I can add in a new commit if desired.

Reply:
@jwhitlock Is there more to `lru_cache` than adding the `@lru_cache.lru_cache()` decorator?

Reply:
It could be that easy, or you may need to do work to make the function arguments hashable. Here's the docs for [lru_cache](https://docs.python.org/3/library/functools.html#functools.lru_cache), which Django backports for Python 2.7.